targets/TARGET_NUVOTON/TARGET_M451/lp_ticker.c

```diff
@@ -115,10 +115,7 @@ void lp_ticker_set_interrupt(timestamp_t timestamp)
     uint32_t cmp_timer = timestamp * NU_TMRCLK_PER_TICK;
     cmp_timer = NU_CLAMP(cmp_timer, TMR_CMP_MIN, TMR_CMP_MAX);
     timer_base->CMP = cmp_timer;
-
-    /* NOTE: When engine is clocked by low power clock source (LXT/LIRC), we need to wait for 3 engine clocks. */
     wait_us((NU_US_PER_SEC / NU_TMRCLK_PER_SEC) * 3);
```

Comment:
Why is this wait still needed? If it's still needed, shouldn't the note/comment still be present?

Reply:
@cmonr It was needed, but the `wait_us` comment was removed inadvertently. In the immediately following commit https://github.com/ARMmbed/mbed-os/pull/6394/commits/ccec9d75d6a37c23df1f4f20f8d1169297a629a5, all `wait_us` comments are moved to the front:
```
/* NOTE: When system clock is higher than timer clock, we need to add 3 engine clock
 *       (recommended by designer) delay to wait for above timer control to take effect. */
```